README.md

```diff
@@ -271,6 +271,7 @@ List of all available rules. The rules ported from `golint` are left unchanged a
 | `redefines-builtin-id`|  n/a   | Warns on redefinitions of builtin identifiers                    |    no    |  no   |
 | `function-result-limit` |  int | Specifies the maximum number of results a function can return    |    no    |  no   |
 | `imports-blacklist`   | []string | Disallows importing the specified packages                     |    no    |  no   |
+| `range-loop-var`      | n/a    | Disallows incorrect uses of range loop variables in closures     |    no    |  no   |
```

Comment:
Rule's name and description do not truly reflect the scope of the rule (i.e. for loop values and defer statements are also checked)